packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -33,7 +34,10 @@
                         :is="titleHtmlTag"
                         v-if="title"
                         :id="uid"
-                        :class="['c-megaModal-title', $style['c-megaModal-title']]"
+                        :class="['c-megaModal-title', $style['c-megaModal-title'], {
+                            [$style['c-megaModal-hasBackBtn']]: hasBackButton,
+                            [$style['c-megaModal-modeRTL']]: hasModeRightToLeft
```

Comment:
Same for this one – I'd put this onto the `c-megaModal` element.

Also, `isModeRightToLeft` feels more grammatically correct I think for the prop name.